src/components/FilterSidebar/components/TemporalFilter/styles.scss

```diff
@@ -20,3 +20,8 @@
     }
   }
 }
+
+// Remove duplicate header in react-datepicker
+.react-datepicker__current-month {
+  display: none;
+}
```

Comment:
I don't like this as a solution, but having looked at the docs again, I don't have a better answer, even the combined `showMonthYearDropdown` has the parent selector to select previous/next month. I'm minded to say to leave that in as from a UX point of view, going to the previous/next month is a single click, whereas in this solution it is 2 clicks.

I note that we are setting 2 classes on the datepicker that were there against the old, I'm not sure they are needed and in fact the `start-date` one doesn't even exist. I can see a case for saying that shouldn't come under this ticket, but I can also say it should. We can chat on this point.

Reply:
Removed this styling